### DISCUSSION TITLE
Fundamentos C#

### DISCUSSION THREAD

**Discussion by luanhenriquee** [balta-io/forum#443] [Tira Dúvidas] [answered]
Olá professor, o mini curso está na versão 3.1, porém muitas coisas o vscode não tem mais suporte para essa versão, como por exemplo o debugger.. Eu irei conseguir acompanhar o curso utilizando a versão mais recente? 8.0

**Comment by andrebaltieri** [comment, marked as answer]
@luanhenriquee bom dia! Irá sim, no código não mudou muita coisa.
Vamos refazer o curso de C# em breve 🚀
